Topics/Product_Management.md

```diff
@@ -12,3 +12,45 @@ Some key features of a successful product manager:
  
  This article goes into a little bit further detail from a beginner's perspective on steps to take to become a successful product manager: [Product Manager: The role and best practices for beginners](https://www.atlassian.com/agile/product-management/product-manager)
+
+### Case Studies:
+
+One of the new challenges for project managers is the new possibility of having to deal with projects on a remote setup. The inability to have face-to-face interactions with teammates often results in lack of communication between team members. This obstacle first emerged for product managers during the COVID-19 pandemic and I will now present two case studies on this issue, illustrating how product managers managed to excel despite these new challenging conditions.
+
+#### Case Study 1 - Working Remotely:
+
+Referencing this [article](https://curiouscore.com/resource/4-lessons-for-product-management-in-a-post-pandemic-world/) , Irene is a product manager in Singapore working on cybersecurity products at a B2B telecommunications company. During COVID-19, she had to adjust her style of product management to match the new working conditions. To succeed, she had some main strategies:
+* To keep everyone in the loop about the project’s progress, Irene’s team held weekly 30-min calls so team members could align on the tasks they needed to complete, as well as identify any potential blockers.
+* Irene’s team was able to react swiftly and effectively to the changes brought on by COVID-19 by first reviewing an existing product roadmap they had, which allowed everyone on the team to have a common document for reference as they made their decisions and reviewed timelines.
+
+Irene sees the pandemic as transformative, allowing unprecedented work flexibilit. She finds that the newly added option of working remotely adds value to her work and increases the potential of teams. She believes that by incorporating some of these strategies, many product managers will be able mirror her success.
+
+
+#### Case Study 2 - Working Remotely:
+
+We now report the experience of several product managers working at TopTal remotely, referencing this [article](https://www.toptal.com/product-managers/remote/product-managers-shift-to-remote). Sam Nissinen and Darshan Dave, two product managers at TopTal, tell us how this transition presented challenges due to the difficulty of not being physically present with teams. This required product managers to enhance their communication skills but ultimately led to success. 
+
+Sam and Dave, succeeded at overcoming this new challenge by:
+* Established clear expectations and acknowledging diverse work cultures to build trust.
+* Leveraged digital tools and adapting communication methodologies to maintain productivity and team unity.
```

Comment:
Can you provide examples of some digital tools that they used to maintain productive?